NOpenCL/CommandQueue.cs

```diff
@@ -8,7 +8,10 @@ namespace NOpenCL
     using System;
     using System.ComponentModel;
     using NOpenCL.SafeHandles;
+    using System.Diagnostics;
+    using System.Runtime.InteropServices;
 
+    [DebuggerDisplay("Device = {Device.Name}")]
```

Comment:
That's correct. The merge problem varies in severity from project to project. In my case, I'm used to doing complex merges and have good software to help, so even when things conflict it's rarely a real problem. :smile: